packages/code-studio/src/main/AppMainContainer.tsx

```diff
@@ -415,21 +434,40 @@ export class AppMainContainer extends Component<
     this.sendClearFilter();
   }
 
-  handleDataChange(data: AppDashboardData): void {
+  handleDataChange(data: DashboardData): void {
     const { updateWorkspaceData } = this.props;
 
     // Only save the data that is serializable/we want to persist to the workspace
-    const { closed, filterSets, links } = data;
-    updateWorkspaceData({ closed, filterSets, links });
+    const { closed } = data;
```

Comment:
I actually don't know why we're saving `closed` with the workspace data as well - that's something that should just be saved with the dashboard (for re-opening closed markdown panels, though it'd be useful for re-opening _any_ panel as well).

Reply:
I'm just going to undo this for now as out of scope for this PR. Should include it with persistence